xblockgrademe/xblockgrademe.py

```diff
@@ -25,15 +25,29 @@ def workbench_scenarios():
             ('Xblock GradeMe',
              """<sequence_demo>
                     <xblockgrademe />
-                    <xblockgrademe name="My First XBlock" />
                 </sequence_demo>
              """),
         ]
 
-    name = String(
-        default='Xblock GradeMe',
+    display_name = String(
+        default="'Grade Me' Button",
```

Comment:
Alternatively, you could wrap the string in single quotes and then use double quotes around the quoted term.